### DISCUSSION TITLE
No detailed load balancer setup in documentation for management server

### DISCUSSION THREAD

**Discussion by gobiursus** [apache/cloudstack#9863] [Q&A] [answered]
##### ISSUE TYPE

 * Documentation Report

##### COMPONENT NAME

~~~
Management server and documentation.
~~~

##### CLOUDSTACK VERSION

~~~
Latest. 4.19.1.2
~~~

##### OS / ENVIRONMENT

Debian Stable

##### SUMMARY

The documentation doesn't provide information about details for how to setup load balance for multiple management server. It just brought up some ideas, not detailed setup.

##### EXPECTED RESULTS

~~~
Put step-to-step load balancer setup into the documentation.
~~~

**Comment by kiranchavala** [comment, marked as answer]
@gobiursus 

Please find the documentation for load balancer setup for management server 

https://docs.cloudstack.apache.org/en/4.19.1.1/installguide/management-server/index.html#additional-management-servers

https://docs.cloudstack.apache.org/en/4.19.1.1/adminguide/reliability.html#management-server-load-balancing

**Comment by gobiursus** [comment]
> @gobiursus
> 
> Please find the documentation for load balancer setup for management server
> 
> https://docs.cloudstack.apache.org/en/4.19.1.1/installguide/management-server/index.html#additional-management-servers
> 
> https://docs.cloudstack.apache.org/en/4.19.1.1/adminguide/reliability.html#management-server-load-balancing

Sorry, I meant do we have anything like a HAproxy or Nginx configuration file as example?

**Comment by weizhouapache** [comment]
> > @gobiursus
> > 
> > Please find the documentation for load balancer setup for management server
> > 
> > https://docs.cloudstack.apache.org/en/4.19.1.1/installguide/management-server/index.html#additional-management-servers
> > 
> > https://docs.cloudstack.apache.org/en/4.19.1.1/adminguide/reliability.html#management-server-load-balancing
> 
> Sorry, I meant do we have anything like a HAproxy or Nginx configuration file as example?

You can refer to nginx or haproxy examples.
The lb config for mgmt servers is pretty simple( just create a rule for port 8080)
